test_sets/hmi_ptu.txt

```diff
@@ -0,0 +1,17 @@
+./test_scripts/Policies/HMI_PTU/001_Success_flow.lua
+./test_scripts/Policies/HMI_PTU/002_PTU_is_performed_via_app_if_HMI_receives_unsuccessful_responses_from_the_policy_server.lua
+./test_scripts/Policies/HMI_PTU/003_SDL_processes_rpc_from_assigned_group_after_PTU_via_HMI.lua
+./test_scripts/Policies/HMI_PTU/004_SDL_rejects_rpc_from_not_assigned_group_after_PTU_via_HMI.lua
+./test_scripts/Policies/HMI_PTU/005_perform_second_PTU_via_HMI_after_registering_App2.lua
+./test_scripts/Policies/HMI_PTU/006_perform_second_PTU_via_HMI_after_registering_App2_and_first_PTU_via_mobile.lua
+./test_scripts/Policies/HMI_PTU/007_perform_second_PTU_via_mobile_for_App2_after_failed_PTU_via_HMI.lua
+./test_scripts/Policies/HMI_PTU/008_reseting_timeout_after_receiving_OnSystemRequest.lua
+./test_scripts/Policies/HMI_PTU/009_PTU_retry_sequence_HMI_has_timeout_more_then_SDL_ext.lua
+./test_scripts/Policies/HMI_PTU/010_PTU_retry_sequence_HMI_has_timeout_more_then_SDL_propr.lua
+./test_scripts/Policies/HMI_PTU/011_UPDATING_status_after_PolicyUpdate_response.lua
+./test_scripts/Policies/HMI_PTU/012_UPDATING_status_absence_after_erroneous_PolicyUpdate_response.lua
+./test_scripts/Policies/HMI_PTU/013_UPDATING_status_absence_after_timeout_expired_for_PolicyUpdate_response.lua
+./test_scripts/Policies/HMI_PTU/014_Successful_PTU_after_failed_retry_consequtive_trigger_ext.lua
+./test_scripts/Policies/HMI_PTU/015_Successful_PTU_after_failed_retry_consequtive_trigger_propr.lua
+;./test_scripts/Policies/HMI_PTU/016_Successful_PTU_after_failed_retry_parallel_trigger_ext.lua https://github.com/smartdevicelink/sdl_core/issues/3136
```

Comment:
```suggestion
; Tests 016 and 017 should be omitted until core issue #3136 is fixed
;./test_scripts/Policies/HMI_PTU/016_Successful_PTU_after_failed_retry_parallel_trigger_ext.lua https://github.com/smartdevicelink/sdl_core/issues/3136
```

Reply:
@ShobhitAd , please find update in db42c2a .